pkg/readiness/setup.go

```diff
@@ -26,6 +26,21 @@ import (
 // provided Manager object.
 // NOTE: Must be called _before_ the manager is started.
 func SetupTracker(mgr manager.Manager, mutationEnabled bool, externalDataEnabled bool) (*Tracker, error) {
+	tracker, err := SetupTrackerNoReadyz(mgr, mutationEnabled, externalDataEnabled)
+	if err != nil {
+		return nil, err
+	}
+
+	if err := mgr.AddReadyzCheck("tracker", tracker.CheckSatisfied); err != nil {
+		return nil, fmt.Errorf("registering readiness check: %w", err)
+	}
+
+	return tracker, nil
+}
+
+// SetupTracker sets up a readiness tracker and registers it to run under control of the
```

Comment:
Is the comment for SetupTrackerNoReadyz?

Reply:
oops, yep